src/Context.ts

```diff
@@ -50,6 +52,7 @@ class InvocationContext implements Context {
 
   constructor(info: FunctionInfo, request: rpc.IInvocationRequest, logCallback: LogCallback, callback: ResultCallback) {
     this.invocationId = <string>request.invocationId;
+    this.traceContext = CreateTraceContext(request.traceContext);

```

Comment:
Could we do something like we do for `getNormalizedBindingData` and `getBindingDefinitions` instead of creating a new class ExecutionTraceContext class?

Reply:
Changed it to a Dict<any> as discussed. I'm resolving the other comments since they are probably not applicable anymore. Please let me know if I missed anything.